cmd/login/login.go

```diff
@@ -82,7 +82,7 @@ func (r *runner) loginWithCodeName(ctx context.Context, codeName string) error {
 // loginWithURL performs the OIDC login into an installation's
 // k8s api with a happa/k8s api URL.
 func (r *runner) loginWithURL(ctx context.Context, path string, firstLogin bool, tokenOverride string) error {
-	i, err := installation.New(ctx, path, "")
+	i, err := r.commonConfig.GetInstallation(ctx, path, "")
```

Comment:
Instead of creating a new `Installation` instance every time it's needed, instantiate it once, store it in the common config and re-use it.